cell_type_prediction/README.md

```diff
@@ -0,0 +1,56 @@
+
+This framework employs a widely-used strategy for data classification - it uses multpile tools to obtain predictions, combines them and generates a consensus output. More specifically, it generates predictions for novel data using multiple libraries of classifiers trained on existing datasets with high-quality annotations. Predictions for each cell are initially filtered based on tool-specific scores, followed by calculating the frequency of each candidate label as well as [semantic similarity](https://en.wikipedia.org/wiki/Semantic_similarity) across predictions. These metrics allow to ascertain the reliability and agreement of predictions. Finally, a table with 3 top candidate labels, corresponding metrics per cell and datasets of origin is produced. 
+
+## What you will need
+* Expression data in the format of 10X Genomics directory with the following files: 
+    * `matrix.mtx` - gene-by-cell matrix holding expression data 
+    * `genes.tsv` - list of genes (corresponding to rows of the matrix)
+    * `barcodes.tsv` - list of cell IDs (matrix columns)
+* Collection of pre-trained classifiers (generated by [this](https://github.com/ebi-gene-expression-group/cell-types-train-control-workflow) workflow) 
+* Tool evaluation table (produced as output of [this](https://github.com/ebi-gene-expression-group/cell-types-eval-control-workflow) workflow; available [here](https://www.ebi.ac.uk/~a_solovyev/prod_testing_data/tool_perf_pvals.tsv))
+* [Cell Ontology](http://www.obofoundry.org/ontology/cl.html) graph in .obo or .xml format (available [here](https://www.ebi.ac.uk/~a_solovyev/prod_testing_data/cl-basic.obo))
+* Exclusions file in `.yaml` format (needed to filer out unlabelled cells and trivial terms; see example [here](https://www.ebi.ac.uk/~a_solovyev/prod_testing_data/exclusions.yaml))
+* SDRF files for training datasets (required to extract cell ontology terms)
+
+## Configuring and running the workflow 
+The workflow is run in Galaxy. Galaxy is a user-friendly service that provices a GUI for using bioinformatics methods. In your running Galaxy instance, open the workflow called `cell_types_prod_control_wf`. To be precise, this is a workflow-of-workflows that runs predictions from individual tools in a modular manner (see figures below). It consists of the following child workflows: 
+* `garnett_prod_workflow`
+* `scmap_cluster_prod_workflow`
+* `scmap_cell_prod_workflow`
+* `scpred_prod_workflow`
+
+The majority of key parameters are configurable at the level of the outer workflow, however, in cases when the individual tool parameters need to be modified, you can refer to the child workflows' configuration. 
+![Fig.1 Galaxy Workflows](workflows.png)
```

Comment:
Any parameter you think needs to be modified at this point should be modifiable at the top layer, with default values.